extensions/interactions/TextInput/directives/oppia-interactive-text-input.directive.ts

```diff
@@ -59,10 +59,14 @@ angular.module('oppia').directive('oppiaInteractiveTextInput', [
             return ctrl.answer.length > 0;
           };
           ctrl.$onInit = function() {
-            ctrl.placeholder = HtmlEscaperService.escapedJsonToObj(
-              $attrs.placeholderWithValue);
-            ctrl.rows = (
-              HtmlEscaperService.escapedJsonToObj($attrs.rowsWithValue));
+            if ($attrs.placeholderWithValue !== 'undefined') {
```

Comment:
This does not look good. Why can it be undefined? We should not allow it to be undefined in the first place.

Ditto below.

Reply:
Yeah you are right, reverted this.(This was causing trouble when i created question from the admin tab, wont happen in a real world scenario.)